src/main/java/org/spongepowered/api/item/inventory/ItemStack.java

```diff
@@ -131,6 +135,8 @@ static ItemStack of(ItemType itemType, int quantity) {
          */
         Builder itemType(ItemType itemType);
 
+        ItemType getCurrentItem();
```

Comment:
Builders never return their current values as it should never matter. You're building the object, not looking up properties.